src/fidesops/schemas/saas/saas_config.py

```diff
@@ -69,6 +63,16 @@ class Strategy(BaseModel):
     configuration: Dict[str, Any]
 
 
+class Header(BaseModel):
+    name: str
+    value: str
+
+
+class QueryParam(BaseModel):
+    name: str
+    value: str
+
+
 class SaaSRequest(BaseModel):
```

Comment:
Just a thought, while the saas configs are in flux, it might be useful to set extra=Extra.forbid on some of these Pydantic classes under Config https://pydantic-docs.helpmanual.io/usage/model_config/, to help indicate in validation that fields that were once being used are no longer being persisted.

For example, if you pass in a type now, it wouldn't be persisted to the database, but there wouldn't be any validation errors, so a user setting this up wouldn't know that it was not actually being used.

Reply:
Added `extra = Extra.forbid` to `SaaSRequest`

Reply:
This is helpful here, also noting this doesn't propagate to pydantic types inside of SaasRequest, but this is still helpful